artmaker/src/main/java/io/artmaker/composables/ArtMakerDrawScreen.kt

```diff
@@ -191,4 +209,46 @@ internal fun ArtMakerDrawScreen(
             }
         },
     )
+
+    if (shouldShowStylusDialog && state.canShowStylusDialog) {
+        if (stylusDialogType.isEmpty()) return
+        val type = StylusDialogType.valueOf(stylusDialogType)
+        val dialogInfo = when (type) {
+            StylusDialogType.ENABLE_STYLUS_ONLY -> stringResource(R.string.stylus_input_detected_title) to stringResource(R.string.styluse_input_detected_message)
+            StylusDialogType.DISABLE_STYLUS_ONLY -> stringResource(R.string.non_stylus_input_detected_title) to stringResource(R.string.non_stylus_input_detected_message)
+        }
+
+        AlertDialog(
+            icon = { Icon(imageVector = Icons.Filled.Edit, contentDescription = Icons.Filled.Edit.name) },
+            title = { Text(text = dialogInfo.first) },
+            text = { Text(text = dialogInfo.second) },
+            onDismissRequest = { shouldShowStylusDialog = false },
+            confirmButton = {
+                Column(modifier = Modifier.fillMaxWidth(), horizontalAlignment = Alignment.CenterHorizontally) {
+                    Button(onClick = { shouldShowStylusDialog = false }) {
+                        Text(text = stringResource(id = android.R.string.ok))
+                    }
```

Comment:
This button can be removed or the logic to dismiss `shouldShowStylusDialog` updated here. When the user clicks this button, we should dismiss this dialog instead of the dialog popping up again.